[New Product] DuckDb (#8515)
captn3m0/endoflife.date: products/duckdb.md

---
title: DuckDB
addedAt: 2025-09-20
category: database
iconSlug: duckdb
permalink: /duckdb
alternate_urls:

>   - /duck-db

Reviewer: ```suggestion
```

[Could not find](https://www.google.com/search?q=%22duck-db%22) any reference of duckdb under this name.

versionCommand: duckdb version

> releasePolicyLink: https://duckdb.org/2025/09/16/announcing-duckdb-140

Reviewer: ```suggestion
releasePolicyLink: https://duckdb.org/2025/09/16/announcing-duckdb-140.html#long-term-support-lts-edition
```

> changelogTemplate: https://github.com/duckdb/duckdb/releases/tag/__LATEST__

Reviewer: ```suggestion
changelogTemplate: https://github.com/duckdb/duckdb/releases/tag/v__LATEST__
```

eolColumn: Support Status

identifiers:
  - repology: duckdb

> 

Reviewer: Automation based on git tags (`git`) is possible. See https://github.com/endoflife-date/endoflife.date/blob/adffaa1050be03c3d9cc4401e143adc248d76b99/products/apache-couchdb.md?plain=1#L22 for example.

Author: I've just added it, thanks for the example @marcwrobel :pray:

releases:
  - releaseCycle: "1.4"

>     releaseDate: 2025-09-16

Reviewer: ```suggestion
    releaseDate: 2025-09-16
    codename: "Andium"
```

Codename could be added on all releases.

Reviewer: On all releases please.

Author: First time I see this possibility, thanks a lot for pointing it @marcwrobel :rocket:

    eol: 2026-09-16
    lts: true
    latest: "1.4.0"
    latestReleaseDate: 2025-09-16

  - releaseCycle: "1.3"
    releaseDate: 2025-05-21
    eol: 2025-09-16
    latest: "1.3.2"
    latestReleaseDate: 2025-07-07

  - releaseCycle: "1.2"
    releaseDate: 2025-02-05
    eol: 2025-05-21
    latest: "1.2.2"
    latestReleaseDate: 2001-04-07

  - releaseCycle: "1.1"
    releaseDate: 2024-09-09
    eol: 2025-02-05
    latest: "1.1.3"
    latestReleaseDate: 2024-11-02 
    
  - releaseCycle: "1.0"
    releaseDate: 2024-05-29
    eol: 2024-09-09
    latest: "1.0.0"
    latestReleaseDate: 2024-05-29 

  - releaseCycle: "0"
    releaseDate: 2019-06-27
    eol: 2024-05-29
    latest: "0.10.3"
    latestReleaseDate: 2024-05-22
---

> [DuckDB](https://duckdb.org/) is an analytical in-process, serverless SQL database management system.

> Starting with `v1.4.0`, every other DuckDB version is going to be a Long Term Support (LTS) edition.

Reviewer: What about non-LTS releases ?

Author: I understand that for non-LTS ones, only the latest are supported : what's your understanding @marcwrobel ?

Reviewer: That's what they say in  https://duckdb.org/2025/09/16/announcing-duckdb-140.html#long-term-support-lts-edition :+1: 

> In the past, we would automatically deprecate old DuckDB versions whenever the newer version was released. But we’re changing this today.

Reviewer: Non-LTS releases support policy is still not documented.

For LTS DuckDB versions, community support will last a year after the release (for now). DuckDB Labs is also starting
to offer support for older LTS versions after their community support has expired.
This doesn't mean they are supporting [all SQLite releases](https://www.sqlite.org/chronology.html);

> it means they plan as if they will be supporting SQLite until 2050.

Reviewer: Why mentioning SQLite ?

Author: That's a typo and a bad copy/paste (I took sqlit as a bassis for this PR)

Author: Just removed the lines, I did not see them at the very bottom of the `md`

    codename: "Andium"
    eol: 2026-09-16
    lts: true
    latest: "1.4.1"
    latestReleaseDate: 2025-10-07

  - releaseCycle: "1.3"
    releaseDate: 2025-05-21
    codename: "Ossivalis"
    codename: ""
    eol: 2025-09-16
    latest: "1.3.2"
    latestReleaseDate: 2025-07-07

  - releaseCycle: "1.2"
    releaseDate: 2025-02-05
    codename: "Histrionicus"
    eol: 2025-05-21
    latest: "1.2.2"
    latestReleaseDate: 2001-04-07

  - releaseCycle: "1.1"
    releaseDate: 2024-09-09
    codename: "Eatoni"
    eol: 2025-02-05
    latest: "1.1.3"
    latestReleaseDate: 2024-11-02 
    
  - releaseCycle: "1.0"
    releaseDate: 2024-05-29
    codename: "Nivis"
    eol: 2024-09-09
    latest: "1.0.0"
    latestReleaseDate: 2024-05-29 

  - releaseCycle: "0"
    releaseDate: 2019-06-27
    eol: 2024-05-29
    latest: "0.10.3"
    latestReleaseDate: 2024-05-22
---

> [DuckDB](https://duckdb.org/) is an analytical in-process, serverless SQL database management system.

Starting with 1.4, every other DuckDB version is a Long Term Support (LTS) release.
LTS releases are supported for one year.
Non-LTS releases become end-of-life once a newer release (LTS or not) is available.

> [Commercial support by DuckDB Labs](https://duckdblabs.com/) is also available older LTS versions after their community support has expired.

Reviewer: ```suggestion
[Commercial support by DuckDB Labs](https://duckdblabs.com/) is also available for older LTS releases after their community support has expired.
```
